src/com/tigersale/userInterface/inventoryManager/RemoveProductView.java

```diff
@@ -0,0 +1,29 @@
+package com.tigersale.userInterface.inventoryManager;
+
+import com.tigersale.database.ProductTable;
+import com.tigersale.model.InventoryManager;
+import com.tigersale.userInterface.AbstractView;
+
+import java.util.Scanner;
+
+/**
+ * @author James Haller
+ */
+public class RemoveProductView extends AbstractView {
+
+    public RemoveProductView(Scanner scanner) {
```

Comment:
Oh, did I miss the dit products view?  haha, whoops.  Uhm, I think pretty much everything can be changed but the name?  It probably doesnt matter in the scope of this class